stor/tests/test_dx.py

```diff
@@ -533,25 +663,23 @@ class TestStat(DXTestCase):
     def test_stat_folder(self):
         self.setup_temporary_project()
         self.setup_files(['/temp_folder/folder_file.csv'])
-        with pytest.raises(ValueError, match='ending in folders'):
+        with pytest.raises(ValueError, match='Invalid operation'):
             DXPath('dx://'+self.project+':/temp_folder/').stat()
-        with pytest.raises(ValueError, match='ending in folders'):
+        with pytest.raises(exceptions.NotFoundError, match='No data object was found'):
             DXPath('dx://'+self.project+':/temp_folder').stat()
 
     def test_stat_project_error(self):
         self.setup_temporary_project()  # creates project with name in self.project
         test_proj = dxb.DXProject()
-        test_proj.new(self.new_proj_name())  # creates duplicate project
-
-        with pytest.raises(dx.DuplicateProjectError, match='Duplicate projects'):
+        test_proj.new(self.project)  # creates duplicate project
+        self.addCleanup(test_proj.destroy)
+        with pytest.raises(dx.MultipleObjectSameNameError, match='Found more than one project'):
```

Comment:
Pluralization is wrong on this exception btw
```suggestion
        with pytest.raises(dx.MultipleObjectsSameNameError, match='Found more than one project'):
```

Reply:
changed this everywhere